packages/react-core/src/demos/examples/DashboardHeader.js

```diff
@@ -123,20 +124,17 @@ export default class DashboardHeader extends React.Component {
             spacer={{ default: 'spacerNone', md: 'spacerMd' }}
           >
             <ToolbarItem>
-              <Button aria-label="Notifications" variant={ButtonVariant.plain}>
-                <AttentionBellIcon />
-              </Button>
+              <Button aria-label="Notifications" variant={ButtonVariant.plain} icon={<AttentionBellIcon />} />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Button aria-label="App launcher" variant={ButtonVariant.plain} icon={<ThIcon />} />
```

Comment:
In core we use the app launcher component here - that seems to make sense. Though I suspect that should be a menu toggle/menu combo?